### PR TITLE
Add WC Payments welcome page experiment

### LINKED ISSUE #4483: Enable WC Payments menu action for merchants without WC Payments installed
### Description
This is an issue to track the effort on enabling the previous experiment in WooCommerce https://github.com/woocommerce/woocommerce/issues/32219#issuecomment-1002861022 

This time is not an experiment anymore but a feature for all the merchants in countries where WC Payments is supported.

It should be rolled out to stores:
- In WCPay eligible countries.
- Without WCPay installed.
- Not selling CBD, based on WooCommerce onboarding data.
- With marketplace suggestions enabled in WooCommerce. 

For now, the flows will be the same as in the experiments, a sign-up and dismiss flows.

This is the base of the project, with #4529 as a nice-to-have feature if times allow it.

More detail in the project thread: paJDYF-4Ov-p2

### PR DESCRIPTION
**Pull request by ismaeldcom**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:
- Reuse old existing experiment to run a new one according to our project (paJDYF-4Ov-p2).
- Update `register_payments_welcome_page` conditions to match the new ones:
  - WCPay not installed.
  - Live store for at least 90 days.
  - US-based store.
  - No existing WCPay account (by checking account cache).
  - Suggestions not disabled via settings or filter.
  - Page not manually dismissed.
  - User under the experiment (`woocommerce_payments_menu_promo_us_2022`).
- Update page design and copy to meet the new design (paJDYF-4NX-p2).
- Update `wcpay_exit_survey` record event to also handle "Just remove" case, and use snake_case prop names to avoid event not being tracked due to Tracks errors.
- Update promotion ID.

Closes Automattic/woocommerce-payments#4483

#### Setup
Perform the following steps to start meeting all the requirements and have the new Payments menu visible:
- Install this branch version of WC in your local env and be sure to be proxified.
- `woocommerce_admin_install_timestamp` should be at least `now-90days in seconds`, using `1` works.
- **WooCommerce → Settings → Country** must be `US`.
- **WooCommerce → Settings → Advanced → WooCommerce.com** `Enable tracking` and `Show suggestions` must be checked.
- Use [experiment](https://experiments.a8c.com/experiments/20748-woocommerce-payments-menu-promo-us-2022/overview) bookmarklet to manually set `treatment` mode.

#### Requirements
Toggle the following steps to confirm that every condition is working and show/hide the Payment menu:
- Install WCPay.
- Connect to a WCPay account. _You'll need to clear `wcpay_account_data` option to meet the condition again_.
- Update `woocommerce_admin_install_timestamp` to today date.
- Choose another WC country.
- Uncheck `Show suggestions`.
- Filter `woocommerce_allow_marketplace_suggestions` to return `false`.
- Set `wc_calypso_bridge_payments_dismissed` option to `yes`.
- Set experiment to `control`.

#### Promotion
- Checkout WCPay server `add/2368-payments-in-core-incentive-fee` branch.
- Accept the promotion.
- Complete the onboarding process.
- The promotion should be applied, you can check it in **Payments → Overview**.

#### Tracks
- Click on `No thanks`
- Click on `Just remove WooCommerce Payments`
- Do it again but check any option and click on `Remove and send feedback`
- Check Tracks Live View and confirm the events are properly registered [with their props](https://github.com/woocommerce/woocommerce/blob/e0f6e357bfdcecb25e08f2ab47385687a40d55f8/plugins/woocommerce-admin/client/payments-welcome/exit-survey-modal.tsx#L44-L70).

#### Design
It should match the following screenshot.
Screenshot 2022-08-12 at 11 58 29

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [x] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce-admin/client/payments-welcome/index.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ interface activatePromoResponse {
 	success: boolean;
 }
 
-const PROMO_NAME = 'wcpay-promo-2022-3-incentive-100-off';
+const PROMO_NAME = 'wcpay-promo-2022-us-incentive-25-off';
```
Comment (moon0326) [inline plugins/woocommerce-admin/client/payments-welcome/index.tsx]: The text copies say "20% off", but this promo name implies it's 25% off. Is this correct?

**Review by moon0326 — COMMENTED** [review]
Everything looks good to me and tested well 👍 

I've left a comment regarding the discount %.

File: plugins/woocommerce-admin/client/payments-welcome/index.tsx (hunk 1 above)
Comment (ismaeldcom) [reply plugins/woocommerce-admin/client/payments-welcome/index.tsx]: Good catch! We modified the promotion after updating the ID and I forgot to update it again. Addressed in 61a7a35.

**Review by dmallory42 — DISMISSED** [review]
Code looks good and tests and works well! :shipit:

**Review by ilyasfoo — APPROVED** [review]

**Pull request merged**